recipes/pbzip2/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.1.13" %}
+{% set major_version = "1.1" %}
+{% set sha1 = "f61e65a7616a3492815d18689c202d0685fe167d" %}
+{% set md5 = "4cb87da2dba05540afce162f34b3a9a6" %}
+
+package:
+  name: pbzip2
+  version: {{ version }}
+
+source:
+  fn: pbzip2-{{ version }}.tar.gz
+  url: https://launchpad.net/pbzip2/{{ major_version }}/{{ version }}/+download/pbzip2-{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - bzip2
```

Comment:
This particular pinning comes from [here]( https://github.com/conda-forge/conda-forge.github.io/blob/61413ed6f7488be2e52e106f34de9e416aef8a0d/scripts/pin_the_slow_way.py#L33 ).